plasmapy/classes/tests/test_Plasma.py

```diff
@@ -67,10 +69,11 @@ def test_Plasma_derived_vars():
                                 domain_z=np.linspace(0, 1, 1)*u.m)
 
     # Set an arbitrary uniform values throughout the plasma
-    test_plasma.density = 2.0 * np.ones((8, 8)) * u.kg / u.m**3
-    test_plasma.momentum = 10.0 * np.ones((3, 8, 8)) * u.kg / (u.m**2 * u.s)
-    test_plasma.pressure = np.ones((8, 8)) * u.Pa
-    test_plasma.magnetic_field = 0.01 * np.ones((3, 8, 8)) * u.T
+    test_plasma.density[...] = 2.0 *  u.kg / u.m**3
+    test_plasma.momentum[...] = 10.0 * u.kg / (u.m**2 * u.s)
+    test_plasma.pressure[...] = 1 * u.Pa
+    test_plasma.magnetic_field[...] = 0.01 * u.T
+    test_plasma.electric_field[...] = 0.01 * u.V / u.m
```

Comment:
Nice. Hadn't thought of doing it like that but I definitely prefer it to my version.